crates/rspack_binding_options/src/options/raw_builtins/mod.rs

```diff
@@ -65,6 +65,7 @@ pub struct RawMinificationConditions {
 pub struct RawMinification {
   pub passes: u32,
   pub drop_console: bool,
+  pub ascii_only: Option<bool>,
```

Comment:
It would be better if you could change this to `bool` and apply the default value in js side, and it's ok to not change this since I might make a refactor in two or three weeks later

Reply:
ok, i'll handle it